ProjectManagmentSystem/src/Modules/AuthenticationModule/components/ForgetPass/ForgetPass.tsx

```diff
@@ -1,7 +1,88 @@
 import React from 'react'
+import logo from '../../../../Modules/../assets/images/PMS 3.svg'
+import { TextField } from '@mui/material';
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { useNavigate } from "react-router-dom";
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+
+
+
 
 export default function ForgetPass() {
+  const navigate = useNavigate()
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
+
+  const onSubmt = async (data) => {
+    console.log(data)
+    try {
+      const res = await axios.post('https://upskilling-egypt.com:3003/api/v1/Users/Reset/Request', data)
+      console.log(res)
+      toast.success('Your request is being processed, please check your email')
+      navigate('/resetpass')
```

Comment:
The convention with naming URL paths is to make it small letters, and when having multiple words to separate  them with hyphen as the example below from Next.js page. 
![image](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/cc3e8bb4-ff7e-46d0-89c4-acdc929f1891)

The problem with using camelCase or having multiple words together is that search engines will treat these paths as a single word. If we try to select  it, we will find the following:
![URL Naming](https://github.com/Elias-lap/ProjectMangement2/assets/64327685/e17538c9-a6a7-4d60-910a-077cf5648225)